packages/e2e-test-app-fabric/windows/RNTesterApp-Fabric/RNTesterApp-Fabric.cpp

```diff
@@ -0,0 +1,283 @@
+// RNTesterApp-Fabric.cpp : Defines the entry point for the application.
+//
+
+#include "pch.h"
+#include "RNTesterApp-Fabric.h"
+
+#include "../../../../vnext/codegen/NativeDeviceInfoSpec.g.h"
+
+#include <DispatcherQueue.h>
+#include <UIAutomation.h>
+
+#include <winrt/Microsoft.ReactNative.Composition.h>
+#include <winrt/Windows.UI.Composition.Desktop.h>
+
+#include "NativeModules.h"
+#include "ReactPropertyBag.h"
+
+#define MAX_LOADSTRING 100
+#define BTN_ADD 1000
+
+// Work around crash in DeviceInfo when running outside of XAML environment
+// TODO rework built-in DeviceInfo to allow it to be driven without use of HWNDs or XamlApps
```

Comment:
With the Fabric work we're probably due to addressing this TODO rather than copying and pasting it around in different apps.

Reply:
https://github.com/microsoft/react-native-windows/issues/11414 Issue created!